Vagrantfile

```diff
@@ -148,6 +148,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.provider "virtualbox" do |vb, override|
     override.vm.box = "ubuntu/trusty64"
+    # Stop any log type files from being generated.
+    vb.customize [ "modifyvm", :id, "--uartmode1", "disconnected" ]
```

Comment:
Can you put the context in the commit message in a comment here?  I think that'd be more helpful.